advocacy_docs/partner_docs/ThalesCipherTrustManager/04-ConfiguringThalesCipherTrustManager.mdx

```diff
@@ -0,0 +1,91 @@
+---
+title: 'Configuration'
+description: 'Walkthrough on configuring the integration'
+---
+
+Implementing Thales CipherTrust Manager with EDB Postgres Advanced Server 15.2 with TDE requires the following components:
+
+- EDB Postgres Advanced Server version 15.2 
+- Thales CipherTrust Manager 
+
+## Prerequisites
+
+- A running EDB Postgres Advanced Server version 15.2 instance.
+- Thales CipherTrust Manager installed and deployed per your VM environment.
+
+## Configure Thales CipherTrust Manager for EDB Postgres Advanced Server
+
+After you have successfully installed Thales CipherTrust Manager you will then need to follow a few steps so you can create certificates for use.
+
+## Login to Thales CipherTrust Manager UI and Create User
+
+1. Using your static IP address you defined in the Thales CipherTrust Manager setup and configuration, log in to the Thales CipherTrust Manager UI via https and your previously configured Username and Password.
+    
+    ![Thales CipherTrust Manager Login](Images/ThalesCipherTrustLoginScreen.png)
+
+2. You will then navigate to the Access Management section and select `Users`.
+
+3. From here you will select `Add Users` and provide all of the required information:
+- Username
+- Password
+
+4. Click on `Add User`.
+
+    ![Thales CipherTrust Manager Add User](Images/ThalesCipherTrustAddUser.png)
+
+5. Click on the `...` beside the user you just created and selected the View/Edit option.
+
+6. Click on `Expand All` and select/add the following groups for the user:
+- `Key Admins`
+- `Key Users`
+- `All Clients`
+
+7. The user that you created now has the appropriate settings and permissions to view/download keys.
+
+## Create and Download Keys on Thales CipherTrust Manager UI
+
+### Download the CA Key
+
+This is the first in a series of keys that you will need for your KMIP server and pykmip.conf file.
```

Comment:
reword.
You will need the following keys for your KMIP server and ...
In our example we are using
key 1 - description 1
key 2 - description 2
key 3 - description 3
etc.